system_metrics_collector/src/system_metrics_collector/collector.cpp

```diff
@@ -52,6 +52,7 @@ bool Collector::Stop()
 
 void Collector::AcceptData(const double measurement)
 {
+  // todo <dabonnie> should this throw if not started?
```

Comment:
Is this the correct TODO comment? IIRC: I used `// TODO(zmichaels11)` in rosbag2.